### PR TITLE
(feat) KHP-6910: Add deceased details panel to patient-chart

### PR DESCRIPTION
**Pull request by its-kios09**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Screenshots

updateLinking

## Related Issue
[[KHP3-6910] Moved deceased details panel into patient Chart - Jira (atlassian.net)](https://thepalladiumgroup.atlassian.net/browse/KHP3-6910)

## Other

*None.*

[KHP3-6910]: https://thepalladiumgroup.atlassian.net/browse/KHP3-6910?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/esm-patient-clinical-view-app/src/dashboard.meta.tsx [hunk 1]
```diff
@@ -110,3 +109,12 @@ export const caseEncounterDashboardMeta = {
   moduleName: '@kenyaemr/esm-patient-clinical-view-app',
   config: {},
 };
+
+export const morgueDeceasedDetailsDashboardMeta = {
+  slot: 'patient-chart-relationships-slot',
```
Comment (donaldkibet) [inline packages/esm-patient-clinical-view-app/src/dashboard.meta.tsx:114]: Should rename the slot to a more appropriate name

**Review by donaldkibet — CHANGES_REQUESTED** [review]
Thanks @its-kios09.

I have observed some extension being exported form `clinic-view-app` that relates to mogue. I would recommend that all features related to mogue module be handled from the app.

I would also recommend only showing the morgue module for patients that are deceased. IMO it doesn't add any value to show that panel for patient still in care. cc @ojwanganto

File: packages/esm-morgue-app/src/card/avail-compartment.compartment.tsx [hunk 2]
```diff
@@ -16,12 +16,7 @@ interface AvailableCompartmentProps {
 
 const AvailableCompartment: React.FC<AvailableCompartmentProps> = ({ patientInfo, index }) => {
   const { t } = useTranslation();
-  const [count, setCount] = useState(3);
-
-  const handleNavigateToDeceasedDetailsView = () =>
-    navigate({
-      to: window.getOpenmrsSpaBase() + `home/morgue/patient/${patientInfo.uuid}`,
-    });
+  const patientChartUrl = '${openmrsSpaBase}/patient/${patientInfo.uuid}/chart/case-management-encounters';
```
[the remark this answers is not in the document]
Comment (its-kios09) [reply packages/esm-morgue-app/src/card/avail-compartment.compartment.tsx:19]: @donaldkibet i suppose to update the PR with new changes lemme look at it then ping you

File: packages/esm-patient-clinical-view-app/src/dashboard.meta.tsx (hunk 1 above)
Comment (its-kios09) [reply packages/esm-patient-clinical-view-app/src/dashboard.meta.tsx]: Already done

**Comment by its-kios09** [comment]
@donaldkibet I have blocked i have added the condition rendering but the dashboard link disappear but the page appears so, currently I am at WIP

**Comment by donaldkibet** [comment]
cc @its-kios09 please address the conflicts

**Comment by its-kios09** [comment]
> cc @its-kios09 please address the conflicts

sorted

**Review by donaldkibet — APPROVED** [review]

**Pull request merged**